tasks/sealights/sealights-python/0.1/sealights-python.yaml

```diff
@@ -0,0 +1,138 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: sealights-python-instrumentation
+  labels:
+    konflux-ci/sealights: "true"
+    konflux-ci/sealights-language: "python"
+spec:
+  description: >-
+    This Tekton task automates the process of configuring your Python application with Sealights for quality analytics and testing.
+    It retrieves the source code from a trusted artifact, installs Python Sealights agent, configures the app for Sealights using
+    vars from your PipelineRun, scans all .py files, reports scan to Sealights, and stores results to be used later on in testing.
+    The task can be triggered by either Pull Request or other events.
+  results:
+    - name: sealights-bsid
+      type: string
+      description: "A unique identifier generated for the current Sealights build session."
+    - name: sealights-build-name
+      type: string
+      description: "A unique build name generated using the commit SHA and current date to prevent conflicts during test reruns."
```

Comment:
do we need build name after scanning?

Reply:
Maybe for some test iteraction. Will not hurt to have it

Reply:
Ya I don't have a usage for this yet I agree thats its not bad just to have.